packages/components/psammead-sitewide-links/package.json

```diff
@@ -23,14 +23,14 @@
   },
   "homepage": "https://github.com/bbc/psammead/blob/latest/packages/components/psammead-sitewide-links/README.md",
   "dependencies": {
-    "@bbc/gel-foundations": "^0.2.2",
-    "@bbc/psammead-styles": "^0.3.0",
+    "@bbc/gel-foundations": "^0.2.5",
```

Comment:
Can we not pull in `0.3.0` here? I know we dont want to pull `gel-foundations` into simorgh, but we should be fine pulling it in here

Reply:
@dr3 Chris wanted to wait until https://github.com/bbc/psammead/issues/388 had been fixed before pulling in `gel-foundations@0.3.0`

Reply:
drew [10:06 AM]
So https://github.com/bbc/psammead/issues/388 is blocking us fixing the site wide links container? which is visually broken

chrisbashton [10:06 AM]
yep :+1:

drew [10:07 AM]
I think i disagree
>The 0.3.0 of gel-foundations (use of rem over em for breakpoints)
Thats not a bug that needs to be fixed in `gel-foundations`
Our usage of it in grid needs to be fixed
We can use rems in sitewide links, as it pulls in gel-foundations on its own
We can then pull in the new sitewide links component without affecting the page grid

chrisbashton [10:10 AM]
So you disagree that 388 blocks work on the sitewide links component
Makes sense :+1:

Reply:
We should be good to pull it into pssamead, as long as we dont pull it into simorgh

Reply:
Sounds good. Will update this PR.

Reply:
Changes have been made.